client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -490,7 +491,7 @@ static Request update(UpdateRequest updateRequest) throws IOException {
             XContentType upsertContentType = updateRequest.upsertRequest().getContentType();
             if ((xContentType != null) && (xContentType != upsertContentType)) {
                 throw new IllegalStateException("Update request cannot have different content types for doc [" + xContentType + "]" +
-                        " and upsert [" + upsertContentType + "] documents");
```

Comment:
Could you revert these formatting changes that aren't part the change? It make git blame harder to work with and I kind of like the old formatting better anyway.